cluster/examples/kubernetes/ceph/operator-openshift.yaml

```diff
@@ -227,27 +227,17 @@ spec:
         #   value: "role=storage-node; storage=rook, ceph"
         # (Optional) CEPH CSI provisioner tolerations list. Put here list of taints you want to tolerate in YAML format.
         # CSI provisioner would be best to start on the same nodes as other ceph daemons.
-        # - name: CSI_PROVISIONER_TOLERATIONS
-        #   value: |
-        #     - effect: NoSchedule
-        #       key: node-role.kubernetes.io/controlplane
-        #       operator: Exists
-        #     - effect: NoExecute
-        #       key: node-role.kubernetes.io/etcd
-        #       operator: Exists
+        - name: CSI_PROVISIONER_TOLERATIONS
```

Comment:
The provisioner shouldn't need this, right? Only the plugin needs to run on the nodes with the storage client pods.

Reply:
Good catch.